Update SM&GM for INFRA-AZUREAPIMANAGEMENTSERVICE

areina opened the pull request:
### Relevant information

Fix the title of a metric which value is in milliseconds.


### Checklist

* [x] I've read the guidelines and understand the acceptance criteria.
* [x] The value of the attribute marked as `identifier` will be unique and valid. 
* [x] I've confirmed that my entity type wasn't already defined. If it is I'm providing an
 explanation above.

areina commented on definitions/infra-azureapimanagementservice/summary_metrics.yml:
```diff
@@ -11,7 +11,7 @@ requests:
 averageResponseTimeS:
-  title: Average response time (s)
+  title: Average response time (ms)
   unit: SECONDS
```
It isn't clear to me what we should do with the unit when the metric is in milliseconds. Should we divide the value by 1000 in the query and leave the metric in seconds?

naxhh replied:
Yes we should. In fact the UI adapts to whatever it seems fit.

So if the value is 0.05s it will probably show 5ms which makes me thing that the unit in the title is probably not a good thing.. but that's probably for another PR/discussion

srvaroa replied:
My knee-jerk reaction is that it'd be nice for all entity types to report times in the same unit, but at the same time I can see this being impractical (e.g. one entity type having response times where the reasonable unit can be ms, another entity type having a different time metric where ms is too small).  For now it seems ok that we optimize locally to each type. WDYT?

areina replied:
What should we put as unit in that case? I don't think we expose `MILLISECONDS` as a unit and having to specify in the title seems pretty hacky.

gvassallo replied:
it won't always be displayed in ms in the UI as it changes based on the value. e.g. 1000 ms might become 1s. I would not specify the unit in the title

areina commented:
Hey folks, I've updated the PR addressing your feedback and with the last version of the main branch. Would you mind taking a look again, please? Thank you!
